packages/components/label/src/label.tsx

```diff
@@ -36,8 +36,10 @@ export type TLabelProps = {
    * ````
    */
   htmlFor?: string;
-  // Indicates if the label title should be in bold text
+  // Indicates if the label title should be in bold text. This is deprecated, use `fontWeight` instead.
   isBold?: boolean;
+  // Indicates if the label title should be in bold text
```

Comment:
Nit picky: Really this is indicating the level of "boldness", not if it's bold or not.

Reply:
True, I'll correct it